common/lib/dependabot/errors.rb

```diff
@@ -223,6 +223,15 @@ def self.updater_error_details(error)
           "go-mod": error.go_mod
         }
       }
+    when
+      DependencyConflict,
+      RequiredVersionIsNotSatisfied,
```

Comment:
Initially I assumed this error was in reference to a dependency not having a version, but I see below it's specifically in reference to the Node version which is required by the project.

> The current .*Node.* version.*does not satisfy the required version

In that case, we already have an exception `ToolVersionNotSupported`, see how we use it in the Python ecosystem: https://github.com/dependabot/dependabot-core/blob/0f4c2677b4c2002d6dee382590322eaae696488f/python/lib/dependabot/python/language_version_manager.rb#L60-L67

Reply:
Checking it and going to change it with ToolVersionNotSupported.

Reply:
replaced the RequiredVersionIsNotSatisfied with ToolVersionNotSupported